src/themes/creator.js

```diff
@@ -9,696 +9,45 @@ LICENSE file in the root directory of this source tree.
 import type {ThemeT} from '../styles/types.js';
 import type {PrimitivesT} from './types.js';
 import deepMerge from '../utils/deep-merge.js';
-import {getMediaQuery} from '../helpers/responsive-helpers.js';
-
-const WHITE = '#FFFFFF';
-const BLACK = '#000000';
+import colorTokens from './light-theme/color-tokens.js';
+import getColorComponentTokens from './light-theme/color-component-tokens.js';
+import getDeprecatedSemanticColors from './light-theme/color-deprecated-semantic-tokens.js';
+import getTypography from './shared/typography.js';
+import breakpoints from './shared/breakpoints.js';
+import mediaQuery from './shared/media-query.js';
+import sizing from './shared/sizing.js';
+import lighting from './shared/lighting.js';
+import borders from './shared/borders.js';
+import animation from './shared/animation.js';
 
 export default function createTheme(
+  // overrides for colorTokens + primatyFontFamily
   primitives: PrimitivesT,
   overrides?: {},
 ): ThemeT {
+  const {primaryFontFamily, ...rest} = primitives;
```

Comment:
Just calling out that it was a bit difficult to determine what's considered a **primitive** and what's considered a **token**.

Reply:
I agree, although changing the structure of primitives would be a breaking change so we're keeping it as is for now.